### PR TITLE
Task 892486 legend label style for Cartesian, Circular, Funnel, Pyramid and Polar series in MAUI 

### PR DESCRIPTION
**Pull request by Harsha-SF4223**
**Description**
- Added Legend label style and LegendItemCreated sections in legend category for Cartesian, Circular, Funnel, Pyramid and Polar.

### CODE REVIEW

File: MAUI/Cartesian-Charts/Legend.md [hunk 1]
```diff
@@ -104,6 +104,47 @@ this.Content=chart;
 
 {% endtabs %}
 
+## Customizing labels
+
+The [`Label`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html#Syncfusion_Maui_Charts_CartesianSeries_Label) property of [`CartesianSeries`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html) is used to define the label for the corresponding series legend item. The appearance of the label can be customized using the [`LabelStyle`]() property. 
+
+* [`TextColor`]() – used to change the color of the label.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Legend.md:111]: Update the definition like this " Gets or sets the color for the text of the label."

Reference - https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#label-customization

File: MAUI/Cartesian-Charts/Legend.md [hunk 2]
```diff
@@ -104,6 +104,47 @@ this.Content=chart;
...
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Legend>
+        <chart:ChartLegend>
+            <chart:ChartLegend.LabelStyle>
+                <chart:ChartLegendLabelStyle TextColor="Blue" Margin="5" FontSize="18" FontAttributes="Bold" FontFamily="Pacifico-Regular"/>
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Legend.md:125]: change font family type (text is not cleared due this font family type) and update the image based on updated the code

File: MAUI/Cartesian-Charts/Legend.md [hunk 3]
```diff
@@ -324,6 +365,23 @@ this.Content = chart;
 
 ![Legend layout for cartesian chart](Legend-images/cartesian_chart.png)
 
+## Event 
+
+**LegendItemCreated**
+
+The [`LegendItemCreated`]() event is triggered when the chart legend item is created. This argument contains the following information.
+
+* [`LegendItem`]() – Used to customize the label and appearance of individual legend item.
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Legend.md:374]: The text color, font size, icon brush, etc., are obtained from the legend item or chart legend.

If the legend item is used, why is there a need to explain the legend item and other customizable properties in this way?
image

refer this https://help.syncfusion.com/maui/cartesian-charts/axis/types#labelcreated

File: MAUI/Funnel-Charts/Legend.md [hunk 4]
```diff
@@ -82,6 +82,56 @@ this.Content = chart;
...
+* [`FontAttributes`]() - used to change the font style for the legend label. 
+* [`FontSize`]() - used to change the font size for the legend label.
+* [`Margin`]() - used to change the margin size of labels.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<chart:SfFunnelChart ItemsSource="{Binding Data}" 
+                      XBindingPath="Name"
```
Comment (SaiyathAliFathima) [inline MAUI/Funnel-Charts/Legend.md:100]: not added the ItemsSource ,XBindingPath, YBindingPath in cartesian , circular chart , but here added this code snippet

File: MAUI/Polar-Charts/Legend.md [hunk 5]
```diff
@@ -124,6 +124,47 @@ this.Content = chart;
...
+chart.Legend.LabelStyle.FontSize = 18;
+chart.Legend.LabelStyle.FontAttributes = FontAttributes.Bold;
+chart.Legend.LabelStyle.Margin = 5;
+chart.Legend.LabelStyle.FontFamily = "Pacifico-Regular";
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Legend labels customization support in Maui Chart](Legend-images/legend_label_style.png)
```
Comment (SaiyathAliFathima) [inline MAUI/Polar-Charts/Legend.md:166]: image seems cropped

File: MAUI/Pyramid-Charts/Legend.md [hunk 6]
```diff
@@ -82,6 +82,56 @@ this.Content = chart;
...
+* [`FontFamily`]() - used to change the font family for the legend label. 
+* [`FontAttributes`]() - used to change the font style for the legend label. 
+* [`FontSize`]() - used to change the font size for the legend label.
+* [`Margin`]() - used to change the margin size of labels.
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+<chart:SfPyramidChart ItemsSource="{Binding Data}"
```
Comment (SaiyathAliFathima) [inline MAUI/Pyramid-Charts/Legend.md:99]: not added the ItemsSource ,XBindingPath, YBindingPath in cartesian , circular chart , but here added this code snippet

File: MAUI/Cartesian-Charts/Legend.md [hunk 7]
```diff
@@ -324,6 +365,23 @@ this.Content = chart;
...
+* [`TextColor`]() – used to change the color of the label.
+* [`FontFamily`]() - used to change the font family for the legend label. 
+* [`FontAttributes`]() - used to change the font style for the legend label. 
+* [`FontSize`]() - used to change the font size for the legend label.
+* [`Margin`]() - used to change the margin size of labels.
+* [`IconBrush`]() - used to change the color of the legend icon.
+* [`IconType`]() - used to change the icon type for the legend icon.
+* [`IconHeight`]() - used to change the icon height of the legend icon.
+* [`IconWidth`]() - used to change the icon width of the legend icon.
+
```
Comment (SaiyathAliFathima) [inline MAUI/Cartesian-Charts/Legend.md:384]: Address these comments for all chart types legend md files

**Review by SaiyathAliFathima — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/Legend.md (hunk 7 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/Legend.md:383]: updated in all chart types legend

File: MAUI/Polar-Charts/Legend.md (hunk 5 above)
Comment (Harsha-SF4223) [reply MAUI/Polar-Charts/Legend.md:166]: image updated

File: MAUI/Cartesian-Charts/Legend.md (hunk 3 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/Legend.md]: updated content

File: MAUI/Cartesian-Charts/Legend.md (hunk 2 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/Legend.md]: updated type

File: MAUI/Cartesian-Charts/Legend.md (hunk 1 above)
Comment (Harsha-SF4223) [reply MAUI/Cartesian-Charts/Legend.md]: updated definitions

**Review by SaiyathAliFathima — APPROVED** [review]
File changes are fine

**Review by Saravanan-Madhesh — APPROVED** [review]
Approved.

**Pull request merged**